frontend/src/app/services/general-guard/general-guard.service.ts

```diff
@@ -7,7 +7,7 @@ import { AuthService } from '../auth/auth.service';
 @Injectable({
   providedIn: 'root'
 })
-export class AuthGuardService implements CanActivate, CanActivateChild {
+export class GeneralGuardService implements CanActivate, CanActivateChild {
```

Comment:
Why guard located in service folder? It is mistake, guards should locate in folder where are using or in common guards folder